rust/ql/lib/codeql/rust/frameworks/stdlib/lang-core.model.yml

```diff
@@ -1,8 +1,17 @@
 extensions:
+  - addsTo:
+      pack: codeql/rust-all
+      extensible: sourceModel
+    data:
+      - ["lang:core", "crate::mem::zeroed", "ReturnValue.Element", "constant-source", "manual"]
   - addsTo:
       pack: codeql/rust-all
       extensible: summaryModel
     data:
+      # Conversions
+      - ["lang:core", "<_ as crate::convert::Into>::into", "Argument[self].Element", "ReturnValue.Element", "taint", "manual"]
```

Comment:
Could this instead simply go from `Argument[self]` to `ReturnValue`?

Reply:
It would have to be `value` flow for that to work (to include flow with arbitrary access paths that is).  I felt that `.into()` should probably only have taint flow, but the practical upside of getting flow in more cases is strong.  Would you prefer:
```
- ["lang:core", "<_ as crate::convert::Into>::into", "Argument[self]", "ReturnValue", "value", "manual"]
```

Reply:
I see. So tracked content does not flow through `Argument[self]` to `ReturnValue` if the step is a taint step. I was not aware of that. Makes sense as-is then :)

Reply:
It's not a feature of data flow I'm particularly keen on, but it does make a lot of sense in some contexts.